package.json

```
@@ -8,15 +8,16 @@
     "ios": "expo run:ios",
     "lint": "tsc && eslint .",
     "prebuild": "npx expo prebuild && npm run restore-app-icon",
-    "restore-app-icon": "cp -a ./AppIcon.appiconset.backup/. ios/Papillon/Images.xcassets/AppIcon.appiconset/"
+    "restore-app-icon": "cp -a ./AppIcon.appiconset.backup/. ios/Papillon/Images.xcassets/AppIcon.appiconset/",
+    "update": "ncu"
```

mets `"update: "ncu"` en 1ère position pour mettre ncu à l'avant
et je me disais bien que j'avais oublié un argument
```suggestion
    "update": "ncu -u"
```